userdocs/mkdocs.yml

```
@@ -3,7 +3,7 @@ site_description: 'The official CLI for Amazon EKS'
 site_author: 'eksctl authors'
 repo_name: 'weaveworks/eksctl'
 repo_url: 'https://github.com/weaveworks/eksctl'
-copyright: 'Copyright &copy; 2020 Weaveworks'
+copyright: 'Copyright &copy; 2021 Weaveworks'
```

We could also use the current date instead of hardcoding it. I haven't tried it, but it should work: 
```suggestion
copyright: 'Copyright &copy; {{ build_date_utc.strftime('%Y') }} Weaveworks'
```

Reference: https://www.mkdocs.org/user-guide/custom-themes/#build_date_utc

---

oh yeh good idea 👍

---

👍  that doesn't quite work out the box (doesn't get parsed) but I will see if I can get something to work